src/components/Chat/StartChat.tsx

```diff
@@ -106,6 +121,15 @@ const StartChat: React.FC<StartChatProps> = ({ defaultModelName, handleNewChatMe
     })
   }
 
+  const refreshLLMConfigs = async () => {
```

Comment:
this func should also go in llmselectorbutton.tsx

Reply:
Done!